lib/filewatch/discover.rb

```diff
@@ -0,0 +1,141 @@
+require_relative 'watched_file'
+
+module FileWatch
+  class Discover
+    attr_reader  :logger, :files, :wf_vars
```

Comment:
What `wf_vars` mean in this context? Watched files? I prefer to be more verbose.

There is a double space between the `attr_reader` and `:logger`